deploy.Jenkinsfile

```diff
@@ -135,6 +134,36 @@ pipeline {
         }
       }
     }
+
+    stage('Push-Docs') {
+      // Assuming windows
+      steps {
+
+        withCredentials([string(credentialsId: 'GitHub_API_Token',
+                                variable: 'api_token')]) {
+          powershell """
```

Comment:
Could we move this into a `.ps1` script? It's good to have this stuff runnable outside of Jenkins and makes the syntax more simple as we don't need to worry about escaping anything

Reply:
The `version_number` variable can be passed to the script as an argument

Reply:
Is it worth adding a comment mentioning that this is only creating the folder on the GitHub docs and does _not_ update `stable` (could also mention where we update stable).